waltz-common/src/main/java/com/wepay/waltz/common/message/MessageCodecV1.java

```diff
@@ -115,6 +117,16 @@ public Message decode(MessageAttributeReader reader) {
                 }
                 return new CheckStorageConnectivityResponse(reqId, storageConnectivityMap);
 
+            case MessageType.SERVER_PARTITIONS_ASSIGNMENT_REQUEST:
```

Comment:
There is no point updating CodecV1 since the old servers and clients do not understand this.

Reply:
reverted, thanks!